app/views/users/show.html.erb

```diff
@@ -6,6 +6,9 @@
   <% if @chef == current_user %>
     <%= link_to 'Edit profile' %>
   <% end %>
+  <% if @chef == current_user %>
+    <%= link_to 'Delete profile' %>
```

Comment:
This link_to is incomplete. It needs a path to destroy and is also missing data: {turbo_method...... 
The destroy action doesn't work without it

Reply:
ok. will include that aswell!